include/modem/lte_lc.h

```diff
@@ -489,6 +591,19 @@ int lte_lc_func_mode_get(enum lte_lc_func_mode *mode);
  */
 int lte_lc_lte_mode_get(enum lte_lc_lte_mode *mode);
 
+/**@brief Start neighboring cell measurements. It may take a while
+ *	  before the measurement results are ready,
```

Comment:
That's right, only one measurement is performed for each time measurements are started. So `_start` and `_stop` might not be a bit misleading.
How about something like this:
```

/**@brief Initiate a neighboring cell measurement.
 *	  The result of the measurement is reported back as an event of the type
 *	  LTE_LC_EVT_NEIGHBOR_CELLS or LTE_LC_EVT_NEIGHBOR_CELLS_NONE, and a
 *	  event handler must be regsitered to reveive the information.
 *	  Depending on the network conditions and LTE connection state, it may
 *	  take a while before the measurement result is ready and reported back.
 *	  After one of the events is received, the neighbor cell measurements
 *	  are automatically stopped.
 *
 * @return Zero on success or (negative) error code otherwise.
 */
int lte_lc_neighbor_cell_measurement(void);

/**@brief Cancel an ongoing neighbor cell measurement.
 *
 * @return Zero on success or (negative) error code otherwise.
 */
int lte_lc_neighbor_cell_measurement_cancel(void);
```

This is closer to AT docs, although the command for stopping the cell measurements is using "stop" instead of cancel.

Reply:
Looks good, I think that makes the interface more clear.

Reply:
Updated to the suggestion above, with minor tweaks to docs to reflect removal of `_NONE` event and event name change.